device-communication/src/main/java/org/eclipse/hono/communication/core/http/HttpServiceBase.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * ***********************************************************
+ *  Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *  <p>
+ *  See the NOTICE file(s) distributed with this work for additional
+ *  information regarding copyright ownership.
+ *  <p>
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Eclipse Public License 2.0 which is available at
+ *  http://www.eclipse.org/legal/epl-2.0
+ *  <p>
+ *  SPDX-License-Identifier: EPL-2.0
+ * **********************************************************
+ *
+ */
+
+package org.eclipse.hono.communication.core.http;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * Base HTTP service class.
+ */
+public class HttpServiceBase {
```

Comment:
This class is not being used. Can it be removed?

Reply:
Done